data/headerNavLinks.ts

```diff
@@ -6,4 +6,12 @@ const headerNavLinks = [
   { href: '/about', title: 'About' },
 ]
 
+export const headerNavOptions = { 
```

Comment:
No need for an additional field. You can modify the original headerNavLink to something like this:
```
const headerNavLinks = [
  { href: '/', title: 'Home' },
  { href: '/blog', title: 'Blog' },
  { href: '/tags', title: 'Tags' },
  { title: 'Others': children: [
    { href: '/projects', title: 'Projects' },
    { href: '/about', title: 'About' },
  ]
]
```

Reply:
Got it I'll see if i could do that and will change